example/src/logic/uimodel.js

```diff
@@ -81,6 +81,20 @@ class UIModel
         this.hdr = inputObservables.hdrDropped;
 
         this.variant = app.variantChanged$.pipe(pluck("event", "msg"));
+
+        const dropedFileName = inputObservables.gltfDropped.pipe(
+            map( (data) => {
+                return data.mainFile.name;
+            })
+        );
+        dropedFileName.subscribe( (filename) => {
+            if(filename !== undefined)
+            {
+                filename = filename.split('/').pop();
+                filename = filename.substr(0, filename.lastIndexOf('.'));
+                this.app.setSelectedModel(filename);
```

Comment:
can we somehow avoid having specialized functions like these? it should be possible to just set the selection of any dropdown and not need to implement a "setSelected<DropdownName>"

Reply:
maybe "v-model" can be used for that?